### PR TITLE
Changes Related to the Auxiliary Process

### PR DESCRIPTION
**Pull request by raylu-hdf**
This PR is directly related to the PR of the changes in the `utils/vfd_swmr` submodule: https://github.com/HDFGroup/vfd_swmr_util/pull/1/files

The basic process is like this:
1. The auxiliary process is enabled with the option --enable-aux-process for configure.
2. The test script testvfdswmr.sh launches the auxiliary process with the big set test.
3. If --enable-aux-process isn't enabled, the auxiliary process simply skips itself and exits.  The big set test will run by itself.

### CODE REVIEW

File: test/testvfdswmr.sh.in [hunk 1]
```diff
@@ -1083,6 +1083,16 @@ for options in "-d 1" "-d 1 -F" "-d 2 -l 16" "-d 2 -F -l 16" "-d 1 -t" "-d 1 -t
         # first dimension (up to 25 1x16x16)
         #
 
+        # Launch the auxiliary process for testing NFS if the --enable-aux-process option is enabled for configuration.
```
Comment (vchoi-hdfgroup) [inline test/testvfdswmr.sh.in:1086]: Minor: is the indentation for line #1086 to #1095 intentional? or should they be lined up with the rest of the code?

File: test/testvfdswmr.sh.in [hunk 2]
```diff
@@ -1137,6 +1156,17 @@ for options in "-d 1" "-d 1 -F" "-d 2 -l 10" "-d 2 -F -l 10" "-d 1 -t -l 10" "-d
 		continue
 	fi
 	echo launch vfd_swmr_bigset_writer few big, options $options ......may take some time......
+
```
Comment (vchoi-hdfgroup) [inline test/testvfdswmr.sh.in:1159]: Minor: is the indentation for line #1160 to #1169 intentional? or should they be lined up with the rest of the code?

**Review by vchoi-hdfgroup — APPROVED** [review]

File: test/testvfdswmr.sh.in (hunk 1 above)
Comment (raylu-hdf) [reply test/testvfdswmr.sh.in:1086]: Intentional.

File: test/testvfdswmr.sh.in (hunk 2 above)
Comment (raylu-hdf) [reply test/testvfdswmr.sh.in:1159]: Intentional.

**Review by kyang2014 — APPROVED** [review]
I just browsed the code. Everything looks fine to me. The only thing I don't understand is why the github check fails. This is probably expected since I saw Vailin approve the code already.

**Comment by vchoi-hdfgroup** [comment]
I also wondered about the github failing checks.  I guess Ray will check into the failures.

**Comment by raylu-hdf** [comment]
> I also wondered about the github failing checks. I guess Ray will check into the failures.

The failure happened because this PR tried to check out the CMake and Makefile for the auxiliary process but couldn't.  I need to ask Dana why the auxiliary process didn't show up in the submodule after I merged the other PR.

**Pull request merged**